src/main/scala/io/joern/typestubs/Main.scala

```diff
@@ -1,11 +1,71 @@
 package io.joern.typestubs
 
 import io.joern.typestubs.ruby.BuiltinPackageDownloader
+import scopt.OParser
+
+final case class Config(writeToJson: Boolean = false, languageFrontend: String = "all") {
+
+  def writeToJson(value: Boolean): Config = {
+    copy(writeToJson = value)
+  }
+
+  def withLanguageFrontend(value: String): Config = {
+    copy(languageFrontend = value)
+  }
+}
+
+private object Frontend {
+
+  implicit val defaultConfig: Config = Config()
+
+  val cmdLineParser: OParser[Unit, Config] = {
+    val availableFrontendLanguages = Seq("", "ruby", "chsarp", "all")
```

Comment:
```suggestion
    val availableFrontendLanguages = Seq("ruby", "csharp", "all")
```

Reply:
Could these potentially be imported from `CPG`? `io.shiftleft.codepropertygraph.generated.Languages`. Then make a constant for `ALL` so that we don't need to manually type the string out each time.